torchao/quantization/utils.py

```diff
@@ -33,10 +32,8 @@
 
 __all__ = [
     "compute_error",
-    "_apply_logging_hook",
     "quantize_activation_per_token_absmax",
     "quant_int8_dynamic_per_token_linear",
```

Comment:
which other functions in this list can also be privatized? Looks like these 2 at least aren't used outside of torchao (and I don't think we intend them to be). Seems like there are a few more